src/GitHub.VisualStudio/UI/GitHubPane.cs

```diff
@@ -83,17 +88,26 @@ public void Initialize(IServiceProvider serviceProvider)
         {
             if (!initialized)
             {
-                var provider = VisualStudio.Services.GitHubServiceProvider;
-                var teServiceHolder = provider.GetService<ITeamExplorerServiceHolder>();
-                teServiceHolder.ServiceProvider = serviceProvider;
+                InitializeAsync(serviceProvider).Forget();
+            }
+        }
+
+        async Task InitializeAsync(IServiceProvider serviceProvider)
+        {
+            // Allow MEF to initialize its cache asynchronously
+            var asyncServiceProvider = (IAsyncServiceProvider)GetService(typeof(SAsyncServiceProvider));
+            await asyncServiceProvider.GetServiceAsync(typeof(SComponentModel));
 
-                var factory = provider.GetService<IViewViewModelFactory>();
-                viewModel = provider.ExportProvider.GetExportedValue<IGitHubPaneViewModel>();
-                viewModel.InitializeAsync(this).Forget();
+            var provider = VisualStudio.Services.GitHubServiceProvider;
+            var teServiceHolder = provider.GetService<ITeamExplorerServiceHolder>();
+            teServiceHolder.ServiceProvider = serviceProvider;
 
-                View = factory.CreateView<IGitHubPaneViewModel>();
-                View.DataContext = viewModel;
-            }
+            var factory = provider.GetService<IViewViewModelFactory>();
+            viewModel = provider.ExportProvider.GetExportedValue<IGitHubPaneViewModel>();
+            viewModel.InitializeAsync(this).Forget();
```

Comment:
Same here. As I sometimes (flippantly) say around my own team, everywhere you use `Forget()` consider removing the line entirely -- since evidently you don't care whether it succeeds or fails. 😉